### PR TITLE
doc: Update important changes to metrics collection

### PR DESCRIPTION
**Pull request by rnishtala-sumo**
## Purpose of this pull request

Updated important changes in metrics collection
- Histograms dropped (except via instrumentation)
- A reminder to update CRDs for the otel operator.

## Select the type of change

- [ ] Minor Changes - Typos, formatting, slight revisions
- [x] Update Content - Revisions, updating sections
- [x] New Content - New features, sections, pages, tutorials
- [ ] Site and Tools - .clabot, version updates, maintenance, dependencies, new packages for the site (Docusaurus, Gatsby, React, etc.)

### CODE REVIEW

File: docs/send-data/kubernetes/v4/how-to-upgrade.md [hunk 1]
```diff
@@ -78,12 +78,14 @@ kube-prometheus-stack:
             sourceLabels: [__name__]
 ```
 
-#### Apply missing Custom Resource Definition for OpenTelemetry Operator
-
-**When??** If you already use OpenTelemetry Operator in a version below `0.42.0`, apply the following command on your cluster:
+#### Install/Update Custom Resource Definition for OpenTelemetry Operator
```
Comment (jpipkin1) [inline docs/send-data/kubernetes/v4/how-to-upgrade.md:81]:
```suggestion
#### Update custom resource definition for OpenTelemetry operator
```

File: docs/send-data/kubernetes/v4/important-changes.md [hunk 2]
```diff
@@ -19,11 +19,30 @@ As of version 3 of the Chart, Fluent Bit and Fluentd were replaced by the OpenTe
 
 OpenTelemetry cannot collect Prometheus recording rule metrics. The new version therefore stops collecting recording rule metrics and updates will be made to the Kubernetes App to remove these metrics. See the [scraped metrics reference](https://github.com/SumoLogic/sumologic-kubernetes-collection/blob/main/docs/scraped-metrics.md#aggregations-removed) for instructions on how to recover these recording rule metrics in Sumo.
 
+### Drop Histogram metrics
+
+Histogram metrics are not collected as of any version > 4.9.0. One can collect these metrics with an allowlist (regex) using the following key: `sumologic.metrics.allowHistogramRegex`
+
+Note: Histogram metrics could still be collected via auto or manual instrumentation.
```
Comment (jpipkin1) [inline docs/send-data/kubernetes/v4/important-changes.md:26]:
```suggestion
:::note
Histogram metrics could still be collected via auto or manual instrumentation.
:::
```

**Review by jpipkin1 — APPROVED** [review]
Approved, @rnishtala-sumo. I made a few minor changes (see commit "[Updates from review](https://github.com/SumoLogic/sumologic-documentation/pull/4505/commits/f0903308d3576b9b8235a1b48a56d4a06220ccd6)").

**Comment by rnishtala-sumo** [comment]
@jpipkin1 LGTM, thanks!

**Pull request merged**